src/tests/simulation/src/cluster.rs

```diff
@@ -195,6 +195,8 @@ impl Cluster {
                 "192.168.1.1:5690",
                 "--state-store",
                 "hummock+memory-shared",
+                "--parallelism",
+                &conf.compute_node_cores.to_string(),
```

Comment:
To fix the failed test: https://buildkite.com/risingwavelabs/pull-request/builds/13328#0184e806-a28b-4df2-93f5-3d00f8dff702

I debugged locally and found the core number obtained by `default_parallelism` is 1 instead of 2. The cause might be that the `ComputeNodeOpts::parse_from(...)` happened outside the simulation 🤣